src/test_helper.bash

```diff
@@ -91,6 +91,21 @@ supportRiscVCGo() {
   versionGTE "$(go version | awk '{print $3}' | sed 's/^go//')" "1.16"
 }
 
+supportLoongArch() {
+  if [ -f /etc/debian_version ]; then
+    if grep "sid" /etc/apt/sources.list 2>/dev/null >/dev/null; then
+      return 0
+    else
+      return 1
+    fi
+  fi
+  return 0
+}
+
+supportLoongArchGo() {
+  go version | grep -E "1.19|1.20|1.21" >/dev/null 2>&1
```

Comment:
This will not work with Go 1.22 and later.

I think you can just drop this function, as Go <= 1.18 has already reached EOL.

Reply:
ACK